DockDoor/Utilities/WindowUtil.swift

```diff
@@ -102,9 +124,12 @@ enum WindowUtil {
         config.height = Int(window.frame.height * scaleFactor) / Int(Defaults[.windowPreviewImageScale])
 
         config.showsCursor = false
-        config.captureResolution = .best
```

Comment:
This is kind of important, why is that removed?

Reply:
`'captureResolution' is only available in macOS 14.0 or newer`
So we need to condition it to avoid compiler error...
Maybe I'll separate it into a separate "captureWindowModern" function and mark it all as available(macOS 14.0, *), that would be nicer

Reply:
Yes, keep in mind the option of specifying a capture resolution has existed for a while. I'm not exactly sure how to set it on the legacy screen API though... but it is possible.

Reply:
Yes `.bestResolution`, but I already define it